src/room/TokenSource.ts

```diff
@@ -0,0 +1,312 @@
+import { RoomConfiguration } from '@livekit/protocol';
+import { decodeJwt } from 'jose';
+import log, { LoggerNames, getLogger } from '../logger';
+
+const ONE_SECOND_IN_MILLISECONDS = 1000;
+const ONE_MINUTE_IN_MILLISECONDS = 60 * ONE_SECOND_IN_MILLISECONDS;
```

Comment:
i always find these clearer when expressed as conversion factors more directly, like this:

```suggestion
const MSEC_PER_SEC = 1000;
const MSEC_PER_MIN = 60 * ONE_SECOND_IN_MILLISECONDS;
```